client/src/transaction.rs

```diff
@@ -58,6 +59,10 @@ impl<Message_: Message> Transaction<Message_> {
             extrinsic,
         }
     }
+
+    pub fn hash(self) -> Hash {
```

Comment:
The following is equivalent but more descriptive
```suggestion
    pub fn hash(self) -> TxHash {
```